### ISSUE TITLE
(re)Allow open new tabs from links in always incognito mode

### ISSUE THREAD

**Issue by uazo** [bromite/bromite#1154]
### Bromite version

Version: `91.0.4472.50`
Arch: `arm64`
Android version: `10`
Device model: `gemini`

### Flags

enable features
ActionableContentSettings,AutofillPreventMixedFormsFilling,BlockInsecurePrivateNetworkRequests,BookmarksExportUseSaf,CCTIncognito,EphemeralTabUsingBottomSheet:ephemeral_tab_open_mode/1,InsecureFormSubmissionInterstitial,OmniboxDefaultTypedNavigationsToHttps,PageInfoHistory,PageInfoV2,RelatedSearches:stamp/1Ru,SearchReadyOmnibox,StrictOriginIsolation,TabGridLayoutAndroid:enable_tab_group_auto_creation/false,enable-media-internals,fill-on-account-select

disable features
AllowSyncXHRInPageDismissal,ContextualSearchLongpressResolve,FontAccess,SyncingCompromisedCredentials,TabGroupsAndroid,TabGroupsContinuationAndroid,TabGroupsUiImprovementsAndroid

### Is this bug about the SystemWebView?
No

### Is the bug reproducible with latest version?
Yes

### Can the bug be reproduced with corresponding Chromium version?
No

### Is this bug happening in an incognito tab?
Yes

### Allow ads for the site; is the bug caused by the adblocker?
No

### Is the bug a crash?
No

### Describe the bug

@csagan5 can I ask you why of this?

https://github.com/bromite/bromite/blob/master/build/patches/Add-an-always-incognito-mode.patch#L226-L235

### Steps to reproduce the bug

Long press on a url, the item "open in another tab" is not present

### Expected behavior

allow the user in anonymous browsing to be able to open a new tab from a link

### Screenshots

N/A

**Comment by csagan5** [comment]
@uazo it was like this before: https://github.com/bromite/bromite/blob/90.0.4430.204/build/patches/Add-an-always-incognito-mode.patch#L217-L222

**Comment by uazo** [comment]
ok thanks, I'll fix it

**Comment by csagan5** [comment]
> ok thanks, I'll fix it

Any idea why that condition was there in v90?

**Comment by uazo** [comment]
yes, I checked, it was used to make the "normal" open new disappear and leave only the "incognito" one.
with v91, all code in ChromeContextMenuPopulator.java can be deleted without worry

**Comment by abesty** [comment]
@uazo 

I get the same problem here in

Version: 91.0.4472.50
Arch: arm64
Android version: 11
Device model: Samsung m21

**Comment by csagan5** [comment]
Fixed in `91.0.4472.97`.

**Comment by uazo** [comment]
unfortunately it doesn't seem to work for me, although I don't understand why...

**Comment by abesty** [comment]
@csagan5 it doesn't fix yet

**Comment by csagan5** [comment]
@uazo the fix in #1165 was tested and was working?

**Comment by BL4CKH47H4CK3R** [comment]
The bug is still not fixed which is sad !

**Comment by uazo** [comment]
discovered the mystery... you missed my changes.

https://github.com/bromite/bromite/commit/40c93af283996a13275a5b1a0bdc73891fd56189#diff-5dcd24e27caf99e0c4738acd4970dec17b5b53d1297eb0658d049b83768d68f7

Things that happen...

**Comment by csagan5** [comment]
No idea how it happened; will be fixed in next release.

**Comment by uazo** [comment]
fixed in `91.0.4472.102` https://github.com/bromite/bromite/commit/c5c35b06a4f230f1116cd44be81b6851da68ca0f
